mac/Keyman4MacIM/Keyman4MacIM/ha.lproj/Localizable.strings

```diff
@@ -60,3 +66,6 @@
 
 /* copyright label in the Package Information window */
 "copyright-label" = "Copyright:";
+
+/* message displayed to alert user to need grant accessibility permission */
+"privacy-alert-text" = "To function properly, Keyman requires accessibility features:\n\nGrant access in System Preferences, Security & Privacy.\nRestart your system.";
```

Comment:
This fixes #7475 

@sgschantz may want to keep his code for falling back to macOS English strings for handling future strings...

Reply:
> @sgschantz may want to keep his code for falling back to macOS English strings for handling future strings...

No, please don't! It adds unnecessary complexity to the code and means there are multiple methods of resolving a string. The appropriate fix is what we are doing here -- going through crowdin.